react/features/notifications/actions.js

```diff
@@ -41,6 +44,23 @@ export function setNotificationsEnabled(enabled) {
 }
 
 /**
+ * Queues an error notification for display.
+ *
+ * @param {Object} props - The props needed to show the notification component.
+ * @returns {Object}
+ */
+export function showErrorNotification(props) {
+    const { addSupportLink } = props;
+
+    return showNotification(Notification, {
+        ...props,
+        addSupportLink: typeof addSupportLink === 'undefined'
```

Comment:
I wonder if it's a readability improvement to use disable/hideSupportLink instead. That way it's easier to know the default is for the link to display but the problem is it's a negative instead of a positive.

Reply:
You're right, I don't have a hard opinion on this. @hristoterezov what do you think? Should I change the prop to hideSupportLink, given that the default is shown?

Reply:
I think hide is better. Also I'd handle the undefined use case as default prop value in the component.

@virtuacoplenny Do we have a rule to name the props positive? Is this a good practice? 

If most of the props are named this way or if we have this rule maybe we should think of something else for `addSupportLink` in order to be consistent.

Reply:
There isn't a rule to name a prop positive. I was just thinking of the practice with if/else statements were generally it's better to test the positive case so that you don't have to do the extra effort of negating.